selene-lib/src/rules/unbalanced_assignments.rs

```diff
@@ -113,6 +113,21 @@ fn expression_is_nil(expression: &ast::Expression) -> bool {
     }
 }
 
+fn expression_is_ellipsis(expression: &ast::Expression) -> bool {
+    match expression {
+        ast::Expression::Value{value, ..} => {
+            if let ast::Value::Symbol(symbol) = &**value {
+                *symbol.token_type() == TokenType::Symbol { symbol: Symbol::Ellipse} 
+            } else {
+                false
+            }
+            
+        },
+
+        _ => false
+    }
+}

```

Comment:
Can you collapse this match into another `if let` and do `return *symbol.token_type() == TokenType::Symbol { symbol: Symbol::Ellipse }` when they pass? Then, at the end, have `false` as a fail safe.

Reply:
Do you mean?

```    
if let ast::Expression::Value{value, ..} = expression {
    if let ast::Value::Symbol(symbol) = &**value {
        *symbol.token_type() == TokenType::Symbol { symbol: Symbol::Ellipse }
    } else {
        false
    }
} else {
    false
}

```

Reply:
Yep, though minor style changes:

```rs
if let ast::Expression::Value{value, ..} = expression {
    if let ast::Value::Symbol(symbol) = &**value {
        return *symbol.token_type() == TokenType::Symbol { symbol: Symbol::Ellipse };
    } 
} 

false
```

Reply:
Done!